.gitlab-ci.yml

```diff
@@ -362,7 +362,7 @@ terraform:
   variables:
     <<:                            *default-vars
     # https://releases.hashicorp.com/terraform/
-    TERRAFORM_VERSION:             "1.0.4"
+    TERRAFORM_VERSION:             "1.1.0"
```

Comment:
Why not latest `1.3.1`?

Reply:
I need an image for 1.1.0 for a specific deployment, after I also need to build 1.2.4+

Reply:
Actually my code does work on 1.3.1, have changed to that.